src/components/ScenarioBox.tsx

```diff
@@ -28,18 +39,59 @@ const ScenarioBox: React.FC<ScenarioBoxProps> = ({
           textAlign: 'center',
           display: 'flex',
           flexDirection: 'column',
-          justifyContent: 'space-between',
+          justifyContent: 'center',
+          alignItems: 'center',
           height: '100%',
-          minHeight: '220px',
+          minHeight: '240px'
         }}
-        data-testid={`${transformLink(link)}-input`}
+        data-testid={`${link}-input`}
       >
-        <Typography variant='h6' gutterBottom>
-          {title}
-        </Typography>
-        <Typography variant='body2' paragraph>
-          {description}
-        </Typography>
+        <Box
+          sx={{
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            flex: '0 0 auto'
+          }}
+        >
+          <Typography variant='h6' gutterBottom>
+            {title}
+          </Typography>
+        </Box>
+        <Box sx={{ flex: '0 0 auto' }}>
+          <Typography variant='body2' paragraph>
+            {t('common.difficulty')}:
+          </Typography>
+          <Rating
+            name='read-only'
+            value={rating}
+            icon={<Circle fontSize='small' />}
+            emptyIcon={<Circle fontSize='small' />}
+            max={3}
+            readOnly
+            sx={{
+              color:
+                rating >= 2
+                  ? rating >= 3
+                    ? red[300]
+                    : orange[300]
+                  : green[300]
+            }}
+          />
+        </Box>
+        <Box
+          sx={{
+            overflow: 'hidden',
+            textOverflow: 'ellipsis',
+            flex: '1 0 auto',
+            display: 'flex',
+            justifyContent: 'flex-end',
+            alignItems: 'flex-end'
+          }}
+        >
+          <Typography variant='body2' paragraph>
```

Comment:
Its possible to replace the truncateDescription function by this css:
`    sx={{
            overflow: 'hidden',
            textOverflow: 'ellipsis',
            display: '-webkit-box',
            WebkitLineClamp: '2',
            WebkitBoxOrient: 'vertical'
          }}
`

Reply:
and we can also add the tooltip showing the entire description.